sys/ztimer/convert.c

```diff
@@ -42,7 +42,7 @@ void ztimer_convert_init(ztimer_convert_t *ztimer_convert,
     ztimer_convert_t tmp = {
         .lower = lower,
         .lower_entry = {
-            .callback = (void (*)(void *))ztimer_handler,
+            .callback = (void (*)(void *)) ztimer_handler,
```

Comment:
Note: This suggestion from `uncrustify` is only due to `uncrustify` not being able to reliably tell the unary * operator and the pointer type * apart from the binary * operator.

Casts have to be placed like a unary operator directly in from of the variable. So this was actually correct before and should be left as it is.

Reply:
Thank you! I dropped the commit where these changes were introduced :)